packages/typespec-client-generator-core/src/decorators.ts

```diff
@@ -663,36 +663,60 @@ export const $usage: UsageDecorator = (
   value: EnumMember | Union,
   scope?: LanguageScopes,
 ) => {
-  const isValidValue = (value: number): boolean => value === 2 || value === 4;
+  const isValidValue = (value: number): boolean => {
+    // Allow the new usage values: input(2), output(4), jsonMergePatch(16), multipartFormData(32), json(256), xml(512)
```

Comment:
The comment doesn't line up with the code. Why not just use UsageFlags?

Reply:
I'm assuming this was more because the other options in usage flags are emitter-specific and we don't want to fully expose them yet. I think we can start smaller and then build, the other ones, like json merge patch and multipart form data are really tied to operations, so I think it's better to hide them for now

Reply:
It looks like the latest commit added JSON merge patch and multipart usages.

Reply:
Oh nevermind.